coc/entry_logs.py

```diff
@@ -53,7 +56,10 @@ def __getitem__(self, index: int) -> Union[ClanWarLogEntry, RaidLogEntry]:
         items from the endpoint"""
         try:
             ret = self._init_logs[index]
-            return self._model(data=ret, client=self._client, response_retry=self._response_retry)
+            if type(self._model) == type(RaidLogEntry):
```

Comment:
what does this do? why not isinstance()?

Reply:
It doesn't work 🤷‍♂️ idky that is what I tried first

Reply:
I guess my comment is not relevant anymore because (as you explained) the war log already has the clan in it, so it does make sense to not add the tag there.